surfsense_backend/app/services/connector_service.py

```
@@ -539,78 +598,91 @@ async def search_extension(self, user_query: str, user_id: str, search_space_id:
                 if visit_date:
                     # Format the date for display (simplified)
                     try:
                         # Just extract the date part for display
-                        formatted_date = visit_date.split('T')[0] if 'T' in visit_date else visit_date
+                        formatted_date = (
+                            visit_date.split("T")[0]
+                            if "T" in visit_date
+                            else visit_date
+                        )
                         title += f" (visited: {formatted_date})"
                     except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)